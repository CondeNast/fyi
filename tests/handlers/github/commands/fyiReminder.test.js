const { Application } = require('probot')
const plugin = require('../../../../robot')
const fyiReminderEvent = require('../../../fixtures/fyi-reminder')
const models = require('../../../../models')
const Event = models.Event
const Fyi = models.Fyi
const slack = require('../../../../services/slack')

describe('Arch Bot', () => {
  afterAll(async () => {
    await models.sequelize.close()
  })

  describe('FYI Reminder', () => {
    it('Posts Comment for Issue in Repo Repository', async () => {
      await app.receive(fyiReminderEvent)
      expect(github.issues.createComment).toMatchSnapshot()
    })
  })
})
