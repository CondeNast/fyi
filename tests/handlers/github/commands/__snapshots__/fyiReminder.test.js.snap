// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arch Bot FYI Reminder Posts Comment for Issue in Repo Repository 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      Object {
        "body": "Reminder to complete the requested FYI. cc @gautamarora",
        "number": 1,
        "owner": "choosenearme",
        "repo": "super-guacamole",
      },
    ],
  ],
}
`;
